doc/source/contributing/index.md

```diff
@@ -1,50 +1,60 @@
+
+## Contributing to ArviZ
+The most common way of contributing to the project is sending pull requests on the main [ArviZ Github repository](https://github.com/arviz-devs/arviz). You can contribute to ArviZ in the following ways:
 
-Below we list some examples of contributions that might serve as inspiration.
+### Create an issue
+[Submit issues](https://github.com/arviz-devs/arviz/issues/new/choose) for existing bugs or desired enhancements. Check [Issue reports](https://arviz-devs.github.io/arviz/contributing/issue_reports.html) for details on how to write an issue.
 
-* [Submitting issues](https://github.com/arviz-devs/arviz/issues/new/choose) related to bugs or desired enhancements. Check the details at {doc}`issue_reports`.
-* Translate our website to languages other than English. Check the details at {ref}`translate`
-* Fixing outstanding [issues](https://github.com/arviz-devs/arviz/issues) (bugs) with the existing codebase. They range from low-level software bugs to higher-level design problems.
-* Contributing or improving the {ref}`documentation <homapage>` (`docs`) or examples (`arviz/examples`).
-* Help users submitting issues by making sure their issue is clear and/or providing temporal
-  workarounds for them. Check the details at {ref}`issue_triaging`
-* Review Pull Requests ensuring contributions are well tested and documented, that documentation
-  renders correctly and has no typos or mistakes. Check the details at {ref}`review_prs`
-* Adding new or improved functionality to the existing codebase.
+### Translate ArviZ website
+You can [translate](https://python.arviz.org/en/latest/contributing/translate.html#translate) our website to languages other than english. 
 
-Further information on how to contribute to ArviZ can be found on these specific pages:
+### Review PRs
+[Review pull requests](https://arviz-devs.github.io/arviz/contributing/review_prs.html#review-prs) to ensure that the contributions are well tested and documented. For example, you can check if the documentation renders correctly and has no typos or mistakes. 
+
+### Manage issues
+Helping to manage or triage the open issues can be a great contribution and a great opportunity to learn about the various areas of the project. You can [triage existing issues](https://arviz-devs.github.io/arviz/contributing/issue_triaging.html#issue-triaging) to make them clear and/or provide temporary workarounds for them. 
+
+### Write documentation
+You can contribute to ArviZ’s documentation by either creating new content or editing existing content. For instance, you can add new ArviZ examples. To get involved with the documentation:
+1. Familiarize yourself with the [documentation content structure](https://arviz-devs.github.io/arviz/contributing/content_structure.html) and the [tool chain](https://arviz-devs.github.io/arviz/contributing/doc_toolchain.html) involved in creating the website.
```

Comment:
```suggestion
1. Familiarize yourself with the `{ref}`documentation content structure <content_structure>` and the {ref}`tool chain <doc_toolchain>` involved in creating the website.
```

Also necessary to add `(doc_toolchain)=` at the top of the `doc_toolchain.md` file.

Reply:
Fixed